docs/supplement.qmd

```diff
@@ -14,192 +14,258 @@ output:
-
 We add a small number (0.1) to the mean to avoid an ill-defined negative binomial. This generates a vector of negative binomial dispersion parameters indexed starting at forecast horizon $j=1$.
+--->
+```
 
 ### Probabilistic nowcast generation
 
-Using the dispersion parameters for each forecast horizon, $\phi_j,$ for $j = 1,...D$, we can generate probabilistic nowcast matrices by drawing samples from the negative binomial for the sum over the still missing counts:
+Predictive distributions for $X_{t^*}, \dots, X_{t^* - D + 1}$ are obtained by combining our point nowcasts with the estimated overdispersion parameters in a simple plug-in fashion. Specifically, we set
 
 $$
-X_{t^*-j, >d} \sim NegBin(\mu = \hat{X}_{t^*-j, >d}, \phi = \phi(j))
+X_{t, > t^* - t} \sim \text{NegBin}(\hat{x}_{t, > t^* - t}, \phi_{t^* - t}).
 $$
 
-We can sample for any number of draws, and then use the draws to compute any desired quantiles to summarize the outputs.
+The predictive distribution for $X_{t}$ then results by shifting this distribution by the already known value $x_{t, \leq t^* - t}$.
 
-### Zero-handling approximation {#zero-handling-approximation}
+```{=html}
+<!--
 
-In order to produce a nowcast for a specific reference date, we use the number of already observed cases, $x_{t,\leq_d}$ and the delay PMF, $\pi$ to estimate the expected final count of cases $X_{t} = \sum_{d=1}^D X_{t,d}$. We will assume a simple binomial reporting model to derive an approximation for the expectation $E(X_t | X_{t, \leq d},\pi_\leq d)$.
+Using the dispersion parameters $\phi_j,$ for each forecast horizon $j = 1,...D$, we can generate probabilistic nowcast matrices by drawing samples from the negative binomial for the sum over the still missing counts:
 
-For notational simplicity, we will refer to $X_{t}$ as $N$, to indicate the random variable for the total number of final cases at a particular reference time $t$, and $X_{t,d}$ as $X$ to indicate the random variable for the number of cases at a particular reference time $t$ and delay $d$.
+$$
+X_{t^*-j, >d} \sim NegBin(\mu = \hat{X}_{t^*-j, >d}, \phi = \phi(j))
+$$
 
-We will denote the CDF of the delay distribution at a particular delay $d$, $\sum_{d=0}^d \pi_d$ simply by $p$.
+We can sample for any number of draws, and then use the draws to compute any desired quantiles to summarize the outputs.
+-->
+```
 
-However, this simple calculation is undefined when the number of already observed cases is 0.
+### Zero-handling strategy {#zero-handling-approximation}
 
-The following derivation addresses this practical issue based on assumptions about the underlying distributions for which the observed cases at each reference time and reporting delay are drawn from.\
+As mentioned in [Point nowcast generation](#point-nowcast-generation), we use a modified point nowcasts to deal with zero values in preliminary counts. We here motivate this approach from a Bayesian perspective. To this end we assume that
```

Comment:
This sentence isn't making sense to me as is, let me know if my edit is what you meant.
```suggestion
As mentioned in [Point nowcast generation](#point-nowcast-generation), we use a modified approach to generate point nowcasts to deal with zero values in preliminary counts. We here motivate this approach from a Bayesian perspective. To this end we assume that
```